jinja2cli/cli.py

```diff
@@ -212,11 +212,11 @@ def _parse_env(data):
 }
 
 
-def render(template_path, data, extensions, strict=False):
+def render(template_path, data, extensions, strict=False, includes=[]):
```

Comment:
bad idea to pass list in arguments
explanation here: https://web.archive.org/web/20200221224620/http://effbot.org/zone/default-values.htm
should be something like that: 
```
def render(template_path, data, extensions, strict=False, includes=None):
    includes = [] if includes is None else includes
```